### PR TITLE
create readme

### PR DESCRIPTION
**Pull request by bilobrovets**

### CODE REVIEW

**Review by ShurionUA — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
-# taxi-service
+# Taxi-Service
+
+This program simulates the work of a taxi service with a driver authentication system.
```
Comment (Nick97-git) [inline README.md]:
```suggestion
This program simulates the work of a taxi service.  
```

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+- Display All Cars
+- Display All Manufacturers
+
+### Implementation details:
+This project is based on a 3-tier architecture:  
+1. Data access layer (DAO)  
+2. Application layer (Service)  
+3. Presentation layer (Controllers)
+
+### Technologies used:
```
Comment (Nick97-git) [inline README.md]:
```suggestion
### Used technologies:  
```

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
...
+3. Clone the project to your IDE
+4. Use a script from the file taxi-service\src\main\resources\init_db.sql to create a database
+5. Open the class taxi-service\src\main\java\taxi\util\ConnectionUtil.java and replace the value of the  
+   constants: URL, USERNAME, PASSWORD, JDBC_DRIVER with your data  
+6. Open the file taxi-service\src\main\resources\log4j2.xml and   
+   replace the value of the parameter "fileName" with your absolute path to the log file 
+7. Configure Apache Tomcat (Add New Configuration -> TomCat -> Local -> Fix -> taxi-service:war exploded -> OK)
+8. Run the application
+
+
```
Comment (Nick97-git) [inline README.md]: remove empty lines)

File: README.md [hunk 4]
```diff
@@ -1 +1,43 @@
...
+- JSTL
+- HTML
+- Maven 
+
+### Recommendations for running the application:  
+1. Install Apache Tomcat  
+2. Install MySQL and MySQL Workbench 
+3. Clone the project to your IDE
+4. Use a script from the file taxi-service\src\main\resources\init_db.sql to create a database
+5. Open the class taxi-service\src\main\java\taxi\util\ConnectionUtil.java and replace the value of the
```
Comment (Nick97-git) [inline README.md]:
```suggestion
5. Open the class taxi-service\src\main\java\taxi\util\ConnectionUtil.java and replace values of  
```

File: README.md [hunk 5]
```diff
@@ -1 +1,43 @@
...
+### Recommendations for running the application:  
+1. Install Apache Tomcat  
+2. Install MySQL and MySQL Workbench 
+3. Clone the project to your IDE
+4. Use a script from the file taxi-service\src\main\resources\init_db.sql to create a database
+5. Open the class taxi-service\src\main\java\taxi\util\ConnectionUtil.java and replace the value of the  
+   constants: URL, USERNAME, PASSWORD, JDBC_DRIVER with your data  
+6. Open the file taxi-service\src\main\resources\log4j2.xml and   
+   replace the value of the parameter "fileName" with your absolute path to the log file 
+7. Configure Apache Tomcat (Add New Configuration -> TomCat -> Local -> Fix -> taxi-service:war exploded -> OK)
```
Comment (Nick97-git) [inline README.md]:
```suggestion
7. Configure Apache Tomcat 
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- remove redundant / from urls like /drivers/, /cars/, /manufacturers/

**Comment by Nick97-git** [comment]
I don't see changes according to this comment `remove redundant / from urls like /drivers/, /cars/, /manufacturers/`

**Comment by Nick97-git** [comment]
why did you remove /delete from urls for delete controllers, you need to remove redundant / from urls like `/cars/` exactly

File: src/main/webapp/web.xml [hunk 6]
```diff
@@ -49,7 +49,7 @@
     </servlet>
     <servlet-mapping>
         <servlet-name>allCars</servlet-name>
-        <url-pattern>/cars/</url-pattern>
```
Comment (Nick97-git) [inline src/main/webapp/web.xml:52]: you forgot to remove / from index.jsp too

**Review by Nick97-git — CHANGES_REQUESTED** [review]

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**